keybase.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "domain": "lobste.rs",
+  "display_name": "Lobsters",
+
+  "description": "Computing-focused community centered around link aggregation and discussion",
+
+  "brand_color": "#AC130D",
+
+  "logo": {
+    "svg_black": "https://lobste.rs/small-black-logo.svg",
+    "svg_full": "https://lobste.rs/full-color.logo.svg"
+  },
+
+  "username": {
+    "re": "^[A-Za-z0-9][A-Za-z0-9_-]{0,24}$",
```

Comment:
Please add a comment that this needs to match the User model's validation, and a comment there pointing back here.

Reply:
json doesn't have comments, but i will add some comments to the other locations